rust/worker/src/server.rs

```diff
@@ -96,27 +101,45 @@ impl WorkerServer {
     pub(crate) fn set_system(&mut self, system: System) {
         self.system = Some(system);
     }
-}
 
-#[tonic::async_trait]
-impl chroma_proto::vector_reader_server::VectorReader for WorkerServer {
-    async fn get_vectors(
+    pub(crate) fn try_parse_tracecontext(
```

Comment:
Same comment - should we move this into tracing/ so its usable by any grpc server we write in rust?